cmd/demo/node.go

```diff
@@ -258,14 +262,57 @@ func (n *node) listen() error {
 	if err != nil {
 		return errors.WithMessage(err, "could not start tcp listener")
 	}
-	go n.client.HandleChannelProposals(n)
+	db, err := leveldb.LoadDatabase(config.DatabasePath)
+	if err != nil {
+		return errors.WithMessage(err, "creating/loading database")
+	}
+	persister, err := keyvalue.NewPersistRestorer(db)
+	if err != nil {
+		return errors.WithMessage(err, "creating PersistRestorer")
+	}
+	n.client.EnablePersistence(persister)
```

Comment:
Im not sure about in which order i need to call `Persistence`, `Handle`, `OnNewChannel` and `Listen`.

Reply:
The order should be correct.